### PR TITLE
Implemented minimum_asset_loss in scenario_risk

### PR DESCRIPTION
**Pull request by micheles**
Part of https://github.com/gem/oq-engine/issues/6320. This is not the end since we need to make more consistent the behavior of `minimum_asset_loss` in ebrisk (currently losses are discarded when computing the average asset losses but not when computing the total portfolio losses, thus resulting in discrepancies when comparing the two).

### CODE REVIEW

File: openquake/calculators/ebrisk.py [hunk 1]
```diff
@@ -193,15 +192,11 @@ def pre_execute(self):
...
         self.check_number_loss_curves()
-        mal = {lt: getdefault(oq.minimum_asset_loss, lt)
-               for lt in oq.loss_names}
-        logging.info('minimum_asset_loss=%s', mal)
+        mal = self.param['minimum_asset_loss']
         if (oq.aggregate_by and self.E * A > oq.max_potential_gmfs and
-                any(val == 0 for val in mal.values()) and not
-                sum(oq.minimum_asset_loss.values())):
+                any(val == 0 for val in mal.values())):
             logging.warning('The calculation is really big; you should set '
```
Comment (raoanirudh) [inline openquake/calculators/ebrisk.py:198]: you should set → consider setting

File: openquake/qa_tests_data/scenario_risk/case_1/expected/losses_by_event.csv [hunk 2]
```diff
@@ -7,6 +7,5 @@ event_id,structural,rlz_id
 4,3.42458E+02,0
 5,2.52500E+02,0
 6,4.81053E+02,0
```
Comment (raoanirudh) [inline openquake/qa_tests_data/scenario_risk/case_1/expected/losses_by_event.csv:9]: Should we print the event with zero losses in the exported file? A scenario usually has a limited number of events, so it won't be an issue printing the losses for all events. Suppressing the zero loss events is more useful in the event-based outputs which might have oversized tables.

**Review by raoanirudh — APPROVED** [review]
LGTM, but please see comments on the warning message and the exported loss table.

File: openquake/qa_tests_data/scenario_risk/case_1/expected/losses_by_event.csv (hunk 2 above)
Comment (micheles) [reply openquake/qa_tests_data/scenario_risk/case_1/expected/losses_by_event.csv:9]: The point is that Helen (see the original issue) wants to be able to compare scenario_risk with ebrisk, so we must use the same approach, i.e. discard zero losses events in both calculators. Notice that the "events" table always contains all events and does not discard them at all. It is only the event loss table that can miss events.

Comment (micheles) [reply openquake/qa_tests_data/scenario_risk/case_1/expected/losses_by_event.csv:9]: Also, if most scenario_risk calculation are small, there is no need to set `minimum_asset_loss` and no risk of losing events ;-)

**Pull request merged**